tests/example/amicable/amicable.dl

```diff
@@ -0,0 +1,24 @@
+// Find pairs of amicable numbers up to a limit
```

Comment:
Needs the standard license header. e.g. https://github.com/souffle-lang/souffle/blob/master/tests/semantic/load7/load7.dl